src/views/splash.html

```diff
@@ -0,0 +1,134 @@
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div id="intro-social" class="one unit h">
+                                <h4>
+                                    Social
+                                </h4>
+                                <div>
+                                    <a href="https://www.facebook.com/hackcambridge">
+                                        <img id="splash-facebook-logo" src="{{ asset('images/hc-2018/facebook-logo.svg') }}" alt="Facebook logo" draggable="false"/>
+                                    </a>
+                                    <a href="https://twitter.com/@hack_cambridge">
+                                        <img id="splash-twitter-logo" src="{{ asset('images/hc-2018/twitter-logo.svg') }}" alt="Twitter logo" draggable="false"/>
+                                    </a>
+                                    <a href="https://medium.com/@HackCambridge">
+                                        <img id="splash-medium-logo" src="{{ asset('images/hc-2018/medium-logo.svg') }}" alt="Medium logo" draggable="false"/>
+                                    </a>
+                                    <a href="https://www.instagram.com/hack_cambridge/">
+                                        <img id="splash-instagram-logo" src="{{ asset('images/hc-2018/instagram-logo.svg') }}" alt="Instagram logo" draggable="false"/>
+                                    </a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section id="splash-signup" class="red">
+                <div class="container width">
+                    <div class="one half unit row">
+                        <div class="six unit h"></div>
+                        <div class="seven unit h">
+                            <div class="container">
+                                <div class="one unit row">
+                                    <div class="five unit h">
+                                        <h5>
+                                            Sign up now!<br><span class="secondary">We’ll be in touch soon.</span>
+                                        </h5>
+                                    </div>
+                                </div>
+                                <div class="half unit row">
+                                    <div class="five unit h">
+                                        <form id="splash-signup-form" class="half unit row" method="post" action="some-action">
+                                            <div class="four small unit h">
+                                                <input type="email" name="signup-email" required placeholder="Enter your email address." />
```

Comment:
Same here, the full stop feels wrong in a placeholder

Reply:
I think this is really a point of stylistics. I like the full stop because it makes it clear that the instruction is over.
Compare
`Put that over there
Now put that over there
And do this
And this
And that`
with
`Put that over there.
Now put that over there.
And do this.
And this.
And that.`
I find that the full stops in the second one separate each instruction and make them easier to keep track of, but maybe that's just me.

Reply:
I agree with you in the absence of the form, but in the context of input boxes, I think the separation of the instructions is clear from the borders of the input boxes.  However looking at this again in the context of the "Sign up now!  We'll be in touch soon.", the full stop looks a lot less unusual.